criu/seize.c

```diff
@@ -92,13 +114,24 @@ static int freezer_restore_state(void)
 		return -1;
 	}
 
-	if (write(fd, frozen, sizeof(frozen)) != sizeof(frozen)) {
-		pr_perror("Unable to freeze tasks");
-		close(fd);
+	return fd;
+}
+
+static int freezer_restore_state(void)
+{
+	int fd;
+	int exit_code;
```

Comment:
exit_code is used only in this patter:
````
{
  int exit_code = -1;
  ...
  ...
  exit_code = 0;
out:
  return exit_code;
}
```

In this case, I would rename exit_code to ret;